src/components/screens/monitor/blockDetails/blockDetails.js

```diff
@@ -0,0 +1,119 @@
+import { Link } from 'react-router-dom';
+import React from 'react';
+import { DateTimeFromTimestamp } from '../../../toolbox/timestamp';
+import Box from '../../../toolbox/box';
+import Feedback from '../../../toolbox/feedback/feedback';
+import LiskAmount from '../../../shared/liskAmount';
+import PageLayout from '../../../toolbox/pageLayout';
+import TableRow from '../../../toolbox/table/tableRow';
+import CopyToClipboard from '../../../toolbox/copyToClipboard';
+import routes from '../../../../constants/routes';
+import styles from './blockDetails.css';
+
+const BlockDetails = ({ t, blockDetails }) => {
+  const blockDetailsData = (blockDetails && blockDetails.data.data && blockDetails.data.data[0]) || '';
+  return (
+    <PageLayout>
+      <Box isLoading={blockDetails.isLoading} width="full">
+        <Box.Header>
+          <h1>{t('Block details')}</h1>
+        </Box.Header>
+        {
+          blockDetails.error
+            ? (
+              <Box.Content>
+                <Feedback status="error" show>{t('Failed to load block details.')}</Feedback>
+              </Box.Content>
+            )
+            : (
+              <React.Fragment>
+                <div>
+                  <TableRow className={styles.tableRow}>
```

Comment:
There is `Box.Row` component for which you don't need to override `hover` style. See Box demo: https://github.com/LiskHQ/lisk-hub/blob/dd73b2379df1e96828fdadb2097948b1bcfdcd19/src/components/toolbox/box/demo.js#L44

```suggestion
                  <Box.Row>
```